API.md

```diff
@@ -0,0 +1,518 @@
+      "value": "4732004860fake@example.com",
+      "rank": 1
+    }
+  ],
+  "birthDate": "1977-04-09",
+  "address": [
+    {
+      "line": [
+        "553 Shanahan View"
+      ],
+      "city": "Lake Reynaldaburgh",
+      "state": "New Mexico",
+      "postalCode": "24801-6534"
+    }
+  ],
+  "resourceType": "Patient"
+}
+```
+
+<a name="read-get-obs"/>
+
+### GET `[base]/Observation/[:id]`
+
+Get a monitoree lab result via an id, e.g.:
+
+```json
+{
+  "id": 1,
+  "meta": {
+    "lastUpdated": "2020-05-11T02:01:16+00:00"
+  },
+  "status": "final",
+  "subject": {
+    "reference": "Patient/1"
+  },
+  "effectiveDateTime": "2020-05-11T00:00:00+00:00",
+  "valueString": "positive",
+  "resourceType": "Observation"
+}
+```
+
+<a name="read-get-que"/>
+
+### GET `[base]/QuestionnaireResponse/[:id]`
+
+Get a monitoree daily report via an id, e.g.:
+
+```json
+{
+  "id": 3,
+  "meta": {
+    "lastUpdated": "2020-05-11T01:06:37+00:00"
+  },
+  "status": "completed",
+  "subject": {
+    "reference": "Patient/1"
+  },
+  "item": [
+    {
+      "text": "cough",
+      "answer": [
+        {
+          "valueBoolean": false
+        }
+      ]
+    },
+    {
+      "text": "difficulty-breathing",
+      "answer": [
+        {
+          "valueBoolean": false
+        }
+      ]
+    },
+    {
+      "text": "fever",
+      "answer": [
+        {
+          "valueBoolean": false
+        }
+      ]
+    },
+    {
+      "text": "used-a-fever-reducer",
+      "answer": [
+        {
+          "valueBoolean": false
+        }
+      ]
+    },
+    {
+      "text": "chills",
+      "answer": [
+        {
+          "valueBoolean": false
+        }
+      ]
+    },
+    {
+      "text": "repeated-shaking-with-chills",
+      "answer": [
+        {
+          "valueBoolean": false
+        }
+      ]
+    },
+    {
+      "text": "muscle-pain",
+      "answer": [
+        {
+          "valueBoolean": false
+        }
+      ]
+    },
+    {
+      "text": "headache",
+      "answer": [
+        {
+          "valueBoolean": false
+        }
+      ]
+    },
+    {
+      "text": "sore-throat",
+      "answer": [
+        {
+          "valueBoolean": false
+        }
+      ]
+    },
+    {
+      "text": "new-loss-of-taste-or-smell",
+      "answer": [
+        {
+          "valueBoolean": false
+        }
+      ]
+    }
+  ],
+  "resourceType": "QuestionnaireResponse"
+}
+```
+
+<a name="read-get-all"/>
+
+### GET `[base]/Patient/[:id]/$everything`
+
+Use this route to retrieve a FHIR Bundle containing the monitoree, all their lab results, and all their daily reports.
+
+```json
+{
+  "id": "9e02e507-b1c0-4fda-ac40-73dab0bd7dc9",
+  "meta": {
+    "lastUpdated": "2020-05-11T22:12:22-04:00"
+  },
+  "type": "searchset",
+  "total": 8,
+  "entry": [
+  ],
+  "resourceType": "Bundle"
+}
+```
+
+<a name="create"/>
+
+## Creating
+
+The API supports creating new monitorees.
+
+<a name="create-post-pat"/>
+
+### POST `[base]/Patient`
```

Comment:
FWIW, I posted a full `Patient` resource conforming to USCore that was generated with Synthea, and the server only recorded/returned the limited demographic information that Sara Alert supports.

From http://hl7.org/fhir/http.html#create
> A server SHOULD otherwise accept the resource as submitted when it accepts the create, and return the same content when it is subsequently read.

However, since this is a SHOULD, and Sara Alert has a limited and narrow operational scope, I believe the dropping of Patient information is an appropriate action.

However, the server did NOT respond with the `Location` Header upon successful `POST` (which is a FHIR `shall` requirement).

> The server returns a 201 Created HTTP status code, and SHALL also return a Location header which contains the new Logical Id and Version Id of the created resource version:

Reply:
Added `Location` header